hwangsae/dbus/meson.build

```diff
@@ -13,14 +13,25 @@ dbus_headers = []
 foreach iface: dbus_ifaces
   prefix = 'org.hwangsaeul.Hwangsae1.'
   name = prefix + iface[0] 
-  dbus_sources = gnome.gdbus_codegen(
-    iface[1],
-    sources: name + '.xml',
-    interface_prefix: prefix,
-    namespace: 'HwangsaeDBus',
-  )
-  dbus_headers = dbus_sources[0]
-  sources += dbus_sources[1]
+
+  if meson.version().version_compare('>= 0.46.0')
```

Comment:
Ah, I forgot to inform you. Unlike _Gaeguli_ and _Chamge_, _Hwangsae_ relay and recorder won't support ubuntu 16.04 because it depends on gstreamer >=1.14 (but recommend gstreamer >= 1.16 because of srt plugin).

So we can use `gnome.gdbus_codege()` as well as the latest features of meson.

Reply:
Thanks for clarirying, I will remove this part.